src/p5.dance.js

```diff
@@ -697,11 +693,10 @@ module.exports = class DanceParty {
         sprite.mirrorX(
           this.generatedDancer.shouldMirror(this.getCurrentMeasure()) ? -1 : 1
         );
-        this.p5_.image(
-          this.generatedDancer.graphics,
-          sprite.x - location.x,
-          sprite.y - location.y
```

Comment:
This was `0, 0` when the dancer remained in its starting location, or caused drift when it was moved.  Hardcoding to `0, 0` fixes the latter situation.